### PR TITLE
feature/AT-SSO-UI_Implementation-Structure

### PR DESCRIPTION
**Pull request by Normaillescas29**

### CODE REVIEW

**Review by irvingesp2190 — APPROVED** [review]

**Review by irvingesp2190 — APPROVED** [review]

**Review by Normaillescas29 — COMMENTED** [review]
Done

**Review by pinto1997 — APPROVED** [review]

**Review by irvingesp2190 — APPROVED** [review]

**Comment by Ignacio-jipl97** [comment]
Add routes

File: src/routes/at-sso.routes.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const router = Router();
+
+const { renderIndexAdmin, renderUserForm, addUser} = require('../controllers/admin.controller');
+
+//Admin dashboard
+router.get('/', renderIndexAdmin);
+
+//Add user
+router.get('/user', renderUserForm);
+router.post('/user',addUser);
```
Comment (at-internship) [inline src/routes/at-sso.routes.js:11]: router.post('/user', addUser);

File: src/controllers/admin.controller.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const adminCtrl = {};
+
+// Index
+adminCtrl.renderIndexAdmin = (req, res) => {
+    res.render('hola aqui va el index admin');
+};
+
+//Render user form
+adminCtrl.renderUserForm = (req,res) => {
```
Comment (at-internship) [inline src/controllers/admin.controller.js:9]: adminCtrl.renderUserForm = (req, res) =>

**Review by at-internship — CHANGES_REQUESTED** [review]
Please, take a look at the comments

**Pull request closed without merging**